Branch for english fixes (#456)
akeneo/pim-helpcenter: content/md/assets/full-automatic-pam-migration.md

We provide this migration strategy as the simplest and easiest strategy, but we advise you to take a look at the other migration strategies: [family-by-family migration guide](family-by-family-pam-migration.html).

> We advise you this migration if you have a simple usage of our former PAM:

Reviewer: ```suggestion
We recommend you to use this migration if you have a simple usage of our former PAM:
```

> - **All your assets have the same type** (for example, you manage only images) because all assets will be gathered in a single asset family.

Reviewer: ```suggestion
- **All your assets are from one single type** (for example, you only manage images) because all your assets will be gathered in a single asset family.
```
